README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Build Status](https://travis-ci.org/jallmann/tls-interop.svg?branch=master)](https://travis-ci.org/jallmann/tls-interop)
+[![Build Status](https://travis-ci.org/mozilla/tls-interop.svg?branch=master)](https://travis-ci.org/mozilla/tls-interop)
```

Comment:
You probably want to keep your own version here? (Yeah that's annoying...)

Reply:
Since mozilla/tls-interop is still pointing to travis-ci.org/jallmann, which needs to be changed, I decided to change that here, and change it back after the next merge.
The alternative would be someone changing it directly in mozilla/tls-interop, and then ignoring this line on future pull requests.